23_networks_vlan_nested_proxmox_cloud-init_27-oct-2022/README.md

```diff
@@ -79,71 +79,103 @@ Note: instead of range you can mention particular admissible VLANs, i.e. "bridge
 ---------------------------------------------------------------------------
 #### 7. Make 3 linked clones from the Proxmox template
 * On each clone make the following:
-* Update hostname & IP address
+* Set a new hostname (replace `<new-hostname>` in the command with the value you want to set):
     ```
-    hostnamectl set-hostname <...>
+    hostnamectl set-hostname <new-hostname>
     ```
 * Edit hostname and/or IP in the following configuration files:
+    Save current hostname in a variable:
     ```
-    nano /etc/hosts
+    currentname=$(hostname)
     ```
+    Edit hostname & IP in `/etc/hosts` (replace `<new-hostname>` and `<new-IP>` in the command with the values you want to set):
     ```
-    nano /etc/postfix/main.cf
+    sed -i "s/$currentname/<new-hostname>/;s/$currentname/<new-hostname>/" /etc/hosts
+    sed -i -r 's/(\b[0-9]{1,3}\.){3}[0-9]{1,3}\b'/"<new-IP>"/  /etc/hosts
```

Comment:
the last line seems to me improper, because it will change all available IPs, but we need to change only the IP corresponding to our new hostname, no? Besides, do these (and above) commands assume that we launch them under `root`? If yes, why do we not point this explicitely?

Reply:
We launch these commands under the cloudinit user (login & password set in .env file). These file editing performed by this user does not require sudo as well.